make/kube-configs

```diff
@@ -4,7 +4,16 @@ set -o errexit -o nounset
 
 GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
 
+# TODO: We currently skip memory limits for development purposes
+
+. "${GIT_ROOT}/make/include/registry"
+
 cd "$GIT_ROOT"
 . ${GIT_ROOT}/.envrc
-fissile build kube -k kube/ --use-memory-limits=false \
-	    -D $(echo env/*.env | tr ' ' ',')
+fissile build kube \
+    --kube-output-dir="${GIT_ROOT}/kube" \
+    --defaults-file="$(echo env/*.env | tr ' ' ',')" \
+    --use-memory-limits=false \
+    --docker-registry "${IMAGE_REGISTRY}" \
```

Comment:
Eh, I guess we can live with it and clean up later.

We should move away from included scripts where we can cover things using `.envrc`, so that running `fissile` directly works.